cmd/benchmark.go

```diff
@@ -363,6 +369,31 @@ func rallyCommandAction(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func getPackageNameAndVersion(packageFromRegistry string) (string, string, error) {
+	packageFromRegistryReverse := make([]byte, 0, len(packageFromRegistry))
```

Comment:
Why reversing? Package names [cannot contain `-`](https://github.com/elastic/package-spec/blob/dfdfb179900285e071896f364200df23e7e80a36/spec/integration/manifest.spec.yml#L285), and versions [can](https://github.com/elastic/package-spec/blob/dfdfb179900285e071896f364200df23e7e80a36/spec/integration/manifest.spec.yml#L241).

It looks safer to just [`strings.Cut(packageFromRegistry, "-")`](https://pkg.go.dev/strings#Cut).

If we keep this I would like to have it covered by tests.